Preserve Aspect order in Add Mast

silverailscolo opened the pull request:
Upon user request: display the list of Aspects in combo box and edit mast pane in order as provided by mast appearance map. On complex masts it was hard to configure because aspect display order was random using HashTables.

KenC57 commented:
If there was a way to preserve the order they came out of the aspect file or
the mast file, that would allow for ordering them like the normal way
prototypes order the aspects.

-ken c

bobjacobsen commented:
> If there was a way to preserve the order they came out of the aspect file or
> the mast file, that would allow for ordering them like the normal way
> prototypes order the aspects.

See LinkedHashMap https://docs.oracle.com/javase/8/docs/api/java/util/LinkedHashMap.html

silverailscolo commented:
Removed Alpha sorting. I totally agree with @KenC57 and since I switched to LinkedHashMap, the order as provided by getAspects() is no longer messed up in the AddSignalMast methods so I removed the sorting step.
Sometimes, however, Stop is first, sometimes it's the last item. But we can control that in a definition.

silverailscolo commented:
Travis Headless failed (unrelated)  on:
```
7622[ERROR] Failures: 
7623[ERROR]   Z21SimulatorAdapterTest.tearDown:132 Unexpected ERROR or FATAL messages emitted
```
Travis Graphic failed (unrelated) on:
```
7702[ERROR] Failures: 
7703[ERROR]   BlockBossLogicTest.testSimpleBlockOccupancy:83 "Stuck at 4 so yellow sets green" did not occur in time
7704[ERROR]   DispatcherFrameTest.testShowAndClose:29 File AutoTurnouts
```

KenC57 commented:
Restarted build as same errors as prior report.
